bin/test-run

```diff
@@ -88,8 +88,16 @@ printf "Testing Linkerd version [%s] namespace [%s] k8s-context [%s]\\n" "$linke
 
 exit_code=0
 
-run_test "$test_directory/install_test.go" --linkerd-namespace=$linkerd_namespace || exit_code=$?
 run_upgrade_test "$linkerd_namespace"-upgrade
```

Comment:
missing `|| exit_code=$?` that will get used in the next `if` below?

Reply:
`exit_code` was getting set in the `run_upgrade_test`, but i've moved it to here, probably clearer.